web/src/components/SubscriptionPopup.jsx

```diff
@@ -267,6 +334,48 @@ const DisplayNameDialog = (props) => {
   );
 };
 
+const checkedItem = (
+  <ListItemIcon>
+    <Check />
+  </ListItemIcon>
+);
+
+const NotificationToggle = ({ subscription }) => {
+  const { t } = useTranslation();
+
+  const handleToggleBackground = async () => {
+    try {
+      await subscriptionManager.toggleBackgroundNotifications(subscription);
+    } catch (e) {
+      console.error("[NotificationToggle] Error setting notification type", e);
+    }
+  };
+
+  const unmute = async () => {
+    await subscriptionManager.setMutedUntil(subscription.id, 0);
+  };
+
+  if (subscription.mutedUntil === 1) {
+    return (
+      <MenuItem onClick={unmute}>
+        <ListItemIcon>
+          <NotificationsOff />
+        </ListItemIcon>
+        {t("notification_toggle_unmute")}
+      </MenuItem>
+    );
+  }
+
+  return (
+    <MenuItem>
+      {subscription.webPushEnabled === 1 && checkedItem}
+      <ListItemText inset={subscription.webPushEnabled !== 1} onClick={handleToggleBackground}>
+        {t("notification_toggle_background")}
+      </ListItemText>
+    </MenuItem>
+  );
+};
```

Comment:
As discussed in Discord, I looove the idea of just having one global config option `Background notifications: Yes|No` in the Settings and remove all of these per-subscription toggles. 

That would remove
- The dropdown menu item "Background notifications" for each subscription
- The additional switch in the Subscribe dialog

It would be a local-only setting (for now).

Behavior:
- When enabled (`Background notifications: Yes`), the `v1/webpush` endpoint would be called with all subscribed topics listed.
- When disabled (`Background notifications: No`),  the `v1/webpush` endpoint would be called with an empty array.
- When a new topic is subscribed to (in the SubscribeDialog) and background notifications are enabled, the `v1/webpush` endpoint is called
- On login, background notifications are (for now) not enabled. This can change later when server-side per-device[-type] settings are possible
- On logout, the `v1/webpush` endpoint would be called with an empty array.

Good?